api/validators/GenericRequests.ts

```diff
@@ -18,6 +18,12 @@ export class UuidParam {
   uuid: Uuid;
 }
 
+export class HandleParam {
+  @IsAlphanumeric()
```

Comment:
So tbh I was initially planning to set the default value for user handles to `{firstName}-{lastName}-{hash}` as we discussed, but then I realized that I couldn't use the existing validators for `IsAlphanumeric` if I included a dash and would have to create a custom `IsValidHandle` validator. 

I did try the approach of making a custom validator but couldn't figure out how to return different default error messages depending on whether the length constraint or regex was broken. If you know how to handle an error message for invalid length and invalid characters in the same validator and think it's worth changing, I can switch the default to include the dash. 

I do think that it would improve readability of handles as a bonus.

Reply:
We can split both these validations into two separate validators - the `Length(3,32)` decorator, and a custom `HasValidSymbols` (or some similarly named validator) decorator that checks the dashes/regex.